doto-frontend/src/constants/Themes.js

```diff
@@ -0,0 +1,4 @@
+export const Themes = {
+    light: "light",
+    dark: "dark",
```

Comment:
Can I nit-pick a little bit :).
It would be better if it was LIGHT and DARK to comply with the coding standards on the wiki

Reply:
Late by 7 seconds.

Reply:
@lakshyaoberoi